docs/repositories/commits.md

```diff
@@ -46,7 +46,7 @@ This area displays the quality gate status and an overview of the code quality m
 {% endif %}
 {% if page.meta.page_name == "pull request" %}
     -   **Coverage variation:** Variation of code coverage relative to the target branch
-    -   **Diff coverage:** Code coverage of the lines added or changed by the pull request
+    -   **Diff coverage:** Code coverage of the coverable lines added or changed by the pull request, or `∅` if no coverable lines were changed
```

Comment:
Could we make the `applicable` word show here somehow? What I am thinking is that I would love to have a reference place when people have either of 2 questions?
* What does it mean to be applicable?
* What does the ø means?

What do you think?

Reply:
This makes a lot of sense, yes. We're now referring to the new diff coverage value more consistently as "`∅` (not applicable)". :+1: